src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -160,10 +145,6 @@ private function mapComplexCssProperty(string $property, string $value, \DOMElem
         }
     }
 
-    /**
-     * @param \DOMElement $node node to apply styles to
-     * @param string $value the value of the style rule to map
-     */
```

Comment:
Both these contain information which is not obvious from the method or parameter names.  So I think either they should be retained, or a description for the method should be added.